R/relabel.R

```diff
@@ -1,9 +1,9 @@
 #' Automatically relabel factor levels, collapse as necessary
 #'
 #' @param f A factor.
-#' @param fun A function that is applied to each level. Must accept one
-#'   character argument and return a character vector of the same length as its
-#'   input.
+#' @param fun A function or something coercible to one by [rlang::as_function()]
```

Comment:
I think it's more useful to describe exactly what it can be here, rather than requiring the user to read more technical documentation.